policyengine_us/parameters/gov/states/md/tax/income/deductions/standard/max.yaml

```diff
@@ -3,9 +3,12 @@ metadata:
   label: Maryland maximum standard deduction
   period: year
   unit: currency-USD
+  uprating: gov.irs.uprating
```

Comment:
the uprating should be applied to each individual filing status - all breakdown parameters